test/c/eacsl/not-yet-supported/data_invariant/strong_global.t

```diff
@@ -1,7 +1,11 @@
   $ owi c --e-acsl ./strong_global.c --no-value
-  Assert failure: %{.*}
-  Model:
-    (model
-      (symbol_0 i32))
-  Reached problem!
-  [13]
+  [kernel:annot-error] strong_global.c:4: Warning: unexpected token 'global'
```

Comment:
This error message is suspicious ?

Reply:
Same as above, `global` is not yet made a keyword